### DISCUSSION TITLE
Web vault "select all" option only selects first 500 items

### DISCUSSION THREAD

**Discussion by bobfatherx** [dani-garcia/vaultwarden#1580] [Q&A]
### Subject of the issue
Web vault "select all" option only selects first 500 items

### Your environment (Generated via diagnostics page)
* Bitwarden_rs version: v1.20.0
* Web-vault version: v2.19.0
* Running within Docker: true
* Uses a reverse proxy: true
* IP Header check: true (X-Real-IP)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Install method: Docker image
* Clients used: Web vault, desktop, iOS
* Reverse proxy and version: Haproxy 0.61_1
* MySQL/MariaDB or PostgreSQL version: SQLite 3.33.0

### Steps to reproduce

Load a web vault with more than 500 items, click the settings cog and click "select all."

### Expected behaviour

All items to be selected.

### Actual behaviour

Only the first 500 items are selected.

**Comment by RealOrangeOne** [comment]
Selection is done on the client vault I believe, so I _think_ this is more of an issue upstream than here.

**Comment by bobfatherx** [comment]
Got it, sorry about that. Thanks for your quick response, though! Should I bother the devs on bitwarden/web to see if they can reproduce and fix?

Edit: I see another use with an over-large library has already reported it: https://github.com/bitwarden/web/issues/836
